app/controllers/application_controller.rb

```diff
@@ -1,3 +1,10 @@
 class ApplicationController < ActionController::Base
   # pass
+  before_action :set_environment
```

Comment:
Since there’s actual code in this class now, you should get rid of the `# pass` comment :)